pkg/operator/ceph/cluster/controller.go

```diff
@@ -496,23 +497,50 @@ func (c *ClusterController) checkPVPresentInCluster(drivers []string, clusterID
 	return false, nil
 }
 
-// removeFinalizer removes a finalizer
-func removeFinalizer(client client.Client, name types.NamespacedName) error {
-	cephCluster := &cephv1.CephCluster{}
-	err := client.Get(context.TODO(), name, cephCluster)
+func removeFinalizers(client client.Client, name types.NamespacedName) error {
+	// Remove cephcluster finalizer
+	err := removeFinalizer(client, name, &cephv1.CephCluster{}, "")
```

Comment:
What do you think about directly calling `opcontroller.RemoveFinalizer` and `opcontroller.RemoveFinalizerWithName` in this method? 

`removeFinalizer` method seems to adding one more layer but does not seem to add any value, IMO.

Reply:
Yes, right it doesn't add any meaning, 

But getting the object and calling opcontroller.RemoveFinalizer, for 3 times will make a function much bigger.

Reply:
Still feel that `removeFinalizer` method is redundant.